api/users.rb

```diff
@@ -25,6 +25,11 @@
       ["course_stats.active_flags", :desc],
       ["course_stats.inactive_flags", :desc],
     ]
+  elsif sort_by == "recency"
```

Comment:
Is `recency` a good name? Otherwise it might make sense to the other parameter as well so we have:
- `recent_activity`
- `most_activity`.

Reply:
I think `recency` is good!